docs/articles_en/learn-openvino/llm_inference_guide/genai-guide/genai-use-cases.rst

```diff
@@ -0,0 +1,296 @@
+GenAI Use Cases
+=====================
+
+This article provides several use case scenarios for Generative AI model
+inference. The applications presented in the code samples below
+only require minimal configuration, like setting an inference device. Feel free
+to explore and modify the source code as you need.
+
+
+Using GenAI for Text-to-Image Generation
+#######################################
+
+Examples below demonstrate inference on text-to-image models, like Stable Diffusion
+1.5, 2.1, and LCM, with a text prompt as input. The :ref:`main.cpp <maincpp>`
+sample shows basic usage of the ``Text2ImagePipeline`` pipeline.
+:ref:`lora.cpp <loracpp>` shows how to apply LoRA adapters to the pipeline.
+
+
+.. tab-set::
+
+   .. tab-item:: C++
+      :sync: cpp
+
+      .. tab-set::
+
+         .. tab-item:: main.cpp
+            :name: maincpp
+
+            .. code-block:: cpp
+
+               int32_t main(int32_t argc, char* argv[]) try {
```

Comment:
If you provide `main()` you probably should add `#include`s. Knowing what to include is more important when knowing how to wrap the code into `main()`. Python imports are added below after all.